engine/src/main/java/org/terasology/rendering/dag/stateChanges/BindFBO.java

```diff
@@ -28,7 +28,7 @@
  */
 public final class BindFBO extends AbstractStateChange<String> implements FBOManagerSubscriber {
     private static final Integer DEFAULT_FRAME_BUFFER_ID = 0;
-    private static final String DEFAULT_FRAME_BUFFER_NAME = "";
+    private static final String DEFAULT_FRAME_BUFFER_NAME = "display";
```

Comment:
It's ok for now, but longer term, when nodes can generate their own FBOs, it will be necessary to ensure that nothing can generate an FBO with name "display".